app/main.py

```diff
@@ -2,0 +1,17 @@
+def copy_file(command: str) -> None:
+    parsed_command = command.split()
+    if (
+        len(parsed_command) != 3
+        or parsed_command[1] == parsed_command[2]
+        or parsed_command[0] != "cp"
+    ):
+        return
+
+    with open(parsed_command[1], "r") as current_file, open(
+        parsed_command[2], "w"
+    ) as new_file:
+        new_file.write(current_file.read())
```

Comment:
This is a code efficiency issue. The current implementation reads the entire file into memory before writing it to the new file. This could cause issues if the file is very large. Consider reading and writing the file in chunks to improve memory usage. You could use a loop to read a fixed amount of data (e.g., 4096 bytes) at a time and write it to the new file.